### ISSUE TITLE
[Combo] Component menu remains open after focusout

### ISSUE THREAD

**Issue by jamesuhl** [IgniteUI/igniteui-angular#15838]
## Description
When using keyboard only and a screen reader, the drop-down menu remains open after tabbing past the last item in the dropdown, leaving the menu open. 

## Solution 
Try adding an event handler on the dropdown menu such that the user cannot tab out of an open dropdown. Make sure the focus returns to any element that originally opened the dropdown before the user leaves (tabs away from) the component. 

## Additional context
We discovered a problem when we added the Infragistics combobox inside of a Material modal box. After tabbing through the combobox the focus leaves the combobox and consequentially escapes outside the modal box (thus not letting a keyboard-only user re-enter the modal box). 
 If the combobox  could be programmed in such a way that the dropdown menu closes and returns the focus to the “listbox” element (div with role="listbox") or the button that opened the drop-down, instead of allowing the user to tab out of the combobox from an open dropdown, I believe this will prevent the focus from being able to exit out of an open modal box. 

![Image](https://github.com/user-attachments/assets/2f170408-4d2a-47e5-99ce-3aa5dc6b3e2f)
